### ISSUE TITLE
relation to OIDC spec?

### ISSUE THREAD

**Issue by peterfigure** [mattrglobal/oidc-client-bound-assertions-spec#68]
Wondering what the relation between this proposal and https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html is? They seem more complementary rather than the OIDC spec replacing this spec? 

If so, are there any plans to drive adoption of this spec with the OIDF?

**Comment by bellebaum** [comment]
Probably related is the question whether the specification in this repository is still in development or has been superseded by some other method of issuing verified credentials, so I am quietly injecting myself in this conversation

**Comment by peterfigure** [comment]
@tplooker @AdamJLemmon any comments here?

**Comment by AdamJLemmon** [comment]
Thanks guys. Yes the latest work is happening under the OIDF connect working group and SIOP special interest groups now where this spec has been contributed.
The latest is similar but is an evolution of this spec to align with the above linked draft, which is also under development within the aforementioned groups.
Hope that helps!

**Comment by bellebaum** [comment]
@AdamJLemmon I assume you are referring to this draft?
https://openid.net/specs/openid-connect-claims-aggregation-1_0.html
Please correct me if I am wrong

**Comment by tplooker** [comment]
@bellebaum yes that draft is a work in progress which incorporates many aspects of this draft and is where future revisions will be based from.
